dev/MRTCore/mrtcorerelease.yml

```diff
@@ -0,0 +1,650 @@
+# Universal Windows Platform
+# Build a Universal Windows Platform project using Visual Studio.
+# Add steps that test and distribute an app, save build artifacts, and more:
+# https://aka.ms/yaml
+
+name: $(BuildDefinitionName)-$(date:yyMM).$(date:dd)$(rev:rrr)
+
+#trigger:
+#- master
+
+jobs:
```

Comment:
This file is quite a bit longer than WinUI2's MUX-Release.yml.  Is this duplicating stuff from the CI build?  Can this accept the output of another YML and just do sign and publish?  This is essentially what it looks like the WinUI2 release yml does.

Reply:
we can make improvement to the files later. for now, I'd like to get things started. Without a working pipeline, I'd rather avoid the unnecessary content change for now.

Reply:
Agreed, we should get it working ASAP then make a follow-up change to prune if needed.

Reply:
@EHO-makai  should i proceed or wait?